ci/docker/runtime_functions.sh

```diff
@@ -1442,6 +1413,67 @@ deploy_docs() {
     popd
 }
 
+build_python_docs() {
+    set -ex
+    pushd .
+
+    export CC="ccache gcc"
+    export CXX="ccache g++"
+    make docs SPHINXOPTS=-W USE_MKLDNN=0
+
+    popd
```

Comment:
why are pushd and popd required? does make docs change the directory? it doesn't? what the hack?